### ISSUE TITLE
I want to add full commands 

### ISSUE THREAD

**Issue by RAJU2529** [MicrosoftDocs/windows-itpro-docs#4253]
I want to add the full commands with full location paths as an example.
So can i get permissions , to edit this article?.
Thanking you 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3fc86fa9-c1cc-6b35-d3a3-ae5953953340
* Version Independent ID: 44fcb642-3ac8-cb6a-9b7c-9ce0543c1f02
* Content: [Deploy exploit protection mitigations across your organization](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-exploit-guard/import-export-exploit-protection-emet-xml)
* Content Source: [windows/security/threat-protection/windows-defender-exploit-guard/import-export-exploit-protection-emet-xml.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/threat-protection/windows-defender-exploit-guard/import-export-exploit-protection-emet-xml.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @levinec
* Microsoft Alias: **ellevin**

**Comment by jvsam** [comment]
@officedocsbot assign @jvsam

**Comment by illfated** [comment]
@RAJU2529 : Permissions to edit the public article will not be available to anyone other than Microsoft employed internal web admins belonging to the publishing team. The only thing we non-Microsoft users can do, is to open Pull Requests (referencing issue tickets or otherwise) to suggest changes to the public pages via the Github version of the page. Some of the reasons will include the necessity to verify the relevance and validity of the content, correctness and order of appearance.

**Comment by RAJU2529** [comment]
@illfated. Yes sir that I knew it sir. I asked to create pull request to author

**Comment by illfated** [comment]
Fair enough. That will be taken care of if you just indicate via PR title what the page is about. The Microsoft Docs team agents (@jvsam , @mypil , @e0i or new ones) will make sure that the PR gets assigned to either a) the author or b) the current owner or c) the current content department manager. If none of these are valid options, a chief tech engineer will be consulted to validate the content.

TL;DR : Post regular pull requests as before, but try to improve your PR titles (relevant description of the page and/or topic in the PR).

**Comment by jvsam** [comment]
Hi @RAJU2529 you've mentioned adding exploit protection rules on #4252. Kindly tag me on your pull request (PR) so I can assist in getting it reviewed. Also, please don't forget to reference to this issue. It would also help in the future if you review the [Microsoft Docs contributor guide](https://docs.microsoft.com/contribute/)⁠—if you have time⁠—to familiarize yourself with the process and workflow. Thank you for being an active member of the Microsoft Docs community!

**Comment by jvsam** [comment]
Hi @RAJU2529 since the pull request you've created is for this content page, https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-exploit-guard/enable-exploit-protection, we will be closing this issue instead. Please monitor the updates on #4252 . Thank you.

**Comment by jvsam** [comment]
@officedocsbot close
